examples/utils.mjs

```diff
@@ -0,0 +1,21 @@
+const regexPatterns = [
+    /^\s*export\s*\*\s*from\s*.+\s*;\s*$/gm,
```

Comment:
Do we need to worry about dynamic imports? Also should we allow statements without semi-colons? I guess we enforce this on the linting level?